frontend/utils/time.js

```diff
@@ -44,10 +44,13 @@ export function compareISOTimestamps (a: string, b: string): number {
   return A > B ? 1 : (A < B ? -1 : 0)
 }
 
+// Do not initialize locale here so that it runs on the server (navigator is undefined there)
+let locale
```

Comment:
I do not understand this comment. Nothing in this file runs on the server?

I think you can (and should) initialize it as a constant here.

Reply:
No, it will break. see the last Travis build. I'm rolling back.

Reply:
Ah I see why travis is failing. It's failing because `navigator` isn't defined in mocha.

That's OK, so you can just write this as:

```js
// prevent mocha tests from breaking (navigator is undefined in them)
const locale = (navigator && navigator.languages ? navigator.languages[0] : navigator.language) || 'en-US'
```

Reply:
I'm ok-ish with that solution. Question: Will we support SSR on GI?